### PR TITLE
AnneJu step 7: Create Task controller and views

### PR DESCRIPTION
**Pull request by anneju**
- [x] Task CRUD with views

**Create Task**

Screen Shot 2022-12-13 at 11 18 56

**Show Task**

Screen Shot 2022-12-13 at 11 18 42

**Edit/Update Task**

Screen Shot 2022-12-13 at 11 18 50

**Task List**

Screen Shot 2022-12-13 at 11 18 35

- [x] set root url to lonk to `tasks#index`
- [x] install `rubocop`

### CODE REVIEW

File: myapp/app/views/tasks/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          </td>
+          <td>
+            <%= link_to "Delete", task_path(task), method: 'delete', data: { confirm: 'Are you sure you want to delete?'} %>
+          </td>
+        </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<%= link_to 'New Task', new_task_path %>>
```
Comment (mt-ytaher) [inline myapp/app/views/tasks/index.html.erb]:
```suggestion
<%= link_to 'New Task', new_task_path %>
```

extra `>`

**Comment by mt-ytaher** [comment]
@anneju I have left a comment for one point, other than that everything looks fine.

File: myapp/app/views/tasks/index.html.erb (hunk 1 above)
Comment (anneju) [reply myapp/app/views/tasks/index.html.erb]: oops

**Comment by anneju** [comment]
@taheryasir thanks for spotting that. Updated~

**Comment by mt-ytaher** [comment]
@anneju 
One point for future reference.

This PR has like lot of codes and I know most of it are auto-generated. So it's not an issue for this PR.

But for future reference when you will start working on main project, we should make small PR. 
Have a look into [this document ](https://confluence.rakuten-it.com/confluence/display/~yasir.taher/A+guide+to+create+better+Pull+Request)

Happy Coding!!

**Review by mt-ytaher — APPROVED** [review]
LGTM!!
[LGTM](https://lgtm.lol/i/531)

**Pull request merged**